src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageNamespacesConfiguration.cs

```diff
@@ -27,6 +27,14 @@ public class PackageNamespacesConfiguration
         /// </summary>
         public bool AreNamespacesEnabled { get; }
 
+        /// <summary>
+        /// Tuple with number of sources and total number of entries defined with package namespaces, used for telemetry.
+        /// Item1 is number of sources.
+        /// Item2 is total number of of entries.
+        /// item3 is source names.
+        /// </summary>
+        public Tuple<int, int, List<string>> NamespacesMetrics { get; }
```

Comment:
From an API perspective, I'm not sure this is a great method. 

Good think is I think it might make sense to cut the number of sources, because it'll match the general source number.

Reply:
Removed.